hafta1/hafta1.md

```diff
@@ -510,21 +510,21 @@ GNU/Linux dağıtımlarında böyle bir şey yok arkadaşlar. Paket yöneticisi
 
 Paket yöneticisi yalnızca bir programdır ama dağıtımdan dağıtıma farklılık gösterir, her paket yöneticisi her dağıtımda kullanamazsınız.
 
-Örneğin Debian'dan türemiş olan Ubuntu, APT isimli bir paket yöneticisi kullanır.
+Örneğin Arch'dan türemiş olan Archlinux, PACMAN isimli bir paket yöneticisi kullanır.
 
-Yani Debian tabanlı dağıtımlarda APT paket yöneticisi kullanılır.
+Yani Arch tabanlı dağıtımlarda PACMAN paket yöneticisi kullanılır.
```

Comment:
"Örneğin Archlinux tabanlı dağıtımlarda Pacman isimli paket yöneticisi kullanılır."

Reply:
Bu yukarsındaki cümle ile bu cümleyi birleştirelim.